lib/bundler/cli.rb

```diff
@@ -368,6 +368,8 @@ def add(gem_name)
     method_option "filter-patch", :type => :boolean, :banner => "Only list patch newer versions"
     method_option "parseable", :aliases => "--porcelain", :type => :boolean, :banner =>
       "Use minimal formatting for more parseable output"
+    method_option "filter-dependencies", :type => :boolean, :banner =>
```

Comment:
maybe this should be something like `--only-explicit` ? When I see `filter`, I think I'm going to be able to pass in a filtering function myself

Reply:
Yeah, I wasn't sure about the best name... I tried following the "convention" of the --filter-[major|minor|patch] flags here. But if you want, I can change it to something else.
Whatever gets it merged ;)

Reply:
I just updated the PR with the new name for this option.